src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -128,13 +128,13 @@ public enum LanguageVersion
         /// </summary>
         CSharp8 = 800,
 
-        // When this value is released, update LanguageVersionExtensions in the IDE layer to point to it.
+        // When this value is available in the released NuGet packge, update LanguageVersionExtensions in the IDE layer to point to it.
```

Comment:
I didn't understand this. We've made changes to compiler and IDE in single PRs, so I'm surprised that the IDE would be depending on nuget rather than just project references...
To my knowledge, there is one exception only, which is a code style assembly, which does not get to see the latest compiler APIs.

Reply:
So the CodeStyle package includes the analyzers shared project. That has an analyzer `ConvertSwitchStatementToExpressionDiagnosticAnalyzer` that checks for CSharp9. Because CodeStyle is referencing from NuGet we don't have the language version yet.

We can either wait until it's published, or temporarily exclude that analyzer from CodeStyle?

Reply:
Thanks. Filing a follow-up seems fine.

---
In reply to: [437017956](https://github.com/dotnet/roslyn/pull/44911#discussion_r437017956) [](ancestors = 437017956)